tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -398,3 +403,79 @@ export function assertIdentityLinkedResult(
         parseIdentity(context.sidechainRegistry, decodedLinkResult.identity, aesKey).toString()
     );
 }
+/* 
+    assert vc
+    steps:
+    1. compare vc account with signer
+    2. check vc status should be Active
+    3. compare vc payload hash(blake vc payload) with vc hash
+    4. compare vc index vc payload id
+    5. check vc signature
+    6. compare vc wtih jsonSchema
+
+    TODO: This is incomplete; we still need to further check: https://github.com/litentry/litentry-parachain/issues/1873
+*/
+
+export async function assertVc(context: IntegrationTestContext, signer: Signer, data: Bytes) {
+    const vc = context.api.createType('RequestVCResult', data) as unknown as RequestVCResult;
```

Comment:
The name of this variable might be confusing.
This is not VC, this is the **result**.

```
pub struct RequestVCResult {
	pub account: AccountId,
	pub assertion: Assertion,
	pub vc_index: H256,
	pub vc_hash: H256,
	pub vc_payload: AesOutput,
}
```

You can read vc from `vc_payload` field.

Reply:
thanks! fixed.